ui/imodel-components-react/src/imodel-components-react/timeline/InlineEdit.tsx

```diff
@@ -60,13 +60,13 @@ export class InlineEdit extends React.Component<
 
   private _onKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
     switch (event.key) {
-      case Key.Escape:
+      case Key.Escape.valueOf():
```

Comment:
FYI `itwinjs-core` is also updating `eslint-plugin` version: https://github.com/iTwin/itwinjs-core/pull/6440. The rule thar requires `.valueOf()` [is disabled there](https://github.com/iTwin/itwinjs-core/pull/6440/files#diff-a6bb5b3f3b014d8c32db3ed31e0ad05b5845d399bdb81a820353cfe7b7c9ab34).

Reply:
I see, I'd prefer to get rid of these `valueOf` if possible (i.e. disable the rule if there is no better alternative).
If we decide to disable the rule - does it make sense for the rule to be enabled in `@itwin/eslint-plugin (or we still think that most of the time that will be kept enabled).

Reply:
We had the conversation on the usefulness of the enum comparison rule here: https://github.com/iTwin/eslint-plugin/pull/58#discussion_r1489189212 and decided to leave it in for our list of rules. We decided to disable entirely for core repo since it would be too much work to make changes there. It is your decision if you want it to be enabled or disabled for this repo. If you want it disabled and want me to remove all `.valueOf()` instances, I can do that too. Let me know, thanks

Reply:
I like Paulius arguments and since you've already made the changes, let's keep them.

Reply:
Sounds good thanks!